requirement-generator/fetcher.ts

```diff
@@ -116,7 +118,21 @@ const getAllCourses = async <T extends keyof Course>(
   return courses;
 };
 
-const courseFieldFilter = getCourseFieldFilter(['subject', 'catalogNbr']);
+const courseFieldFilter = getCourseFieldFilter([
```

Comment:
What is the process for retrieving course credits for requirements calculations again?

Reply:
Is it directly from user data?

Reply:
Yes, from user data.